public/pages/Detectors/components/DetectorRulesView/DetectorRulesView.tsx

```diff
@@ -33,7 +33,7 @@ const mapRuleItemToRuleTableItem = (ruleItem: RuleItem): RuleTableItem => {
     description: ruleItem.description,
     source: ruleItem.library,
     ruleId: ruleItem.id,
-    ruleInfo: { ...ruleItem.ruleInfo, prePackaged: ruleItem.library === 'Sigma' },
+    ruleInfo: { ...ruleItem.ruleInfo, prePackaged: ruleItem.library === 'Standard' },
```

Comment:
Should we make a constant called `DEFAULT_LIBRARY` and store that in a constants file and import it here since we are making this change in multiple places.

Reply:
Let me do it in a follow up PR, don't want to block on it for this PR.